test/exometer_SUITE.erl

```diff
@@ -417,7 +417,7 @@ test_status(_Config) ->
      {module, exometer_histogram},
      {status, enabled},
      {cache, 0},
-     {value, []},
+     {value, _},
```

Comment:
Wouldn't it make more sense to actually match the datapoints here? Otherwise, a similar error as before could easily sneak in.